lessons/03-html-lists/index.html

```diff
@@ -0,0 +1,67 @@
+<html>
+    <head>
+        <title>Hogwarts</title>
+     </head>
+     <body>
+         <h1>Hogwarts</h1>
+         <h2>Houses</h2>
+         <ul>
+             <li><strong>Gryffindor</strong></li>
+             <li>Hufflepuff</li>
+             <li>Ravenclaw</li>
+             <li>Slytherin</li>
+             <title>Notable teachers and staff members</title>
```

Comment:
The title tag is only valid inside the `<head>` tag, but it seems like you caught that when you added the `<h3>` below this with the proper title